File: Kattis/Quick Brown Fox.py

def string_to_characters(string):
    return [character for character in string]


n = int(input())
alphabet = []

for letter in (range(97, 97 + 26)):
    alphabet.append(chr(letter))

for x in range(n):

    listed_string = string_to_characters(input())
    missing_letters = []

    for index in range(len(listed_string)):
        listed_string[index] = listed_string[index].lower()

    for letter in alphabet:
        if letter not in listed_string:
            missing_letters.append(letter)

    if len(missing_letters) == 0:
        print("pangram")
    else:
        missing_letters.insert(0, "missing ")
        print(*missing_letters, sep="")

File: Kattis/Quick Brown Fox.cpp
#include <iostream>
#include <string>
#include <set>
#include <vector>
#include <algorithm>
using namespace std;

int main()
{
    int numberOfStrings;
    cin >> numberOfStrings;
    string strIn;
    getline(cin, strIn);

    for (int i = 0; i < (numberOfStrings); i++)
    {
        set<char> alphabet{'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'};
        getline(cin, strIn);
        set<char> hits{};

        for (char letter : strIn)
        {
            alphabet.erase(tolower(letter));
        }

        if (alphabet.size() == 0)
        {
            cout << "pangram\n";
        }
        else
        {
            cout << "missing ";
            for (auto j = alphabet.begin(); j != alphabet.end(); ++j)
            {
                std::cout << *j;
            }

            cout << '\n';
        }
    }
    return 0;
}
